samples/unit-tests/series-pie/styled-mode/demo.js

```diff
@@ -1,3 +1,24 @@
+QUnit.test('Pie borderColor null(#1828)', function (assert) {
```

Comment:
Not related to styled mode, move it to the general `unit-tests/series-pie/pie/demo.js`.